CaloDigi/Realistic/src/BruteForceEcalGapFiller.cc

```diff
@@ -86,6 +86,11 @@ BruteForceEcalGapFiller::BruteForceEcalGapFiller( )  : Processor( "BruteForceEca
 			     _intraModuleFactor,
 			     float (1.0) );
 
+  registerProcessorParameter("applyInterModuleCorrection",
+			     "apply correction for gaps between modules?",
+			     _applyInterModuleCor,
+			     bool(false) );
```

Comment:
Can you set it to true by default ? To avoid implicit change in processor behavior.

Reply:
I have set it to true. 
We have to remember to switch it off in production